.github/workflows/e2e.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
           echo "$SSH_KEY" > "$e2e_ssh_key"
           chmod 600 "$e2e_ssh_key"
           export E2E_SSH_KEY="$e2e_ssh_key"
-          export E2E_SSH_HOST="${{ steps.get_public_dns_name.outputs.result }}"
+          export E2E_SSH_HOST="${{ steps.holodeck_public_dns_name.outputs.result }}"
```

Comment:
Question: Could we set this in the `env` section instead?

Reply:
yes